spec/api-browser-window-spec.ts

```diff
@@ -2102,13 +2102,13 @@ describe('BrowserWindow module', () => {
       });
 
       ifdescribe(process.platform === 'win32')('Fullscreen state', () => {
-        it('with properties', () => {
+        describe('with properties', () => {
           it('can be set with the fullscreen constructor option', () => {
             w = new BrowserWindow({ fullscreen: true });
             expect(w.fullScreen).to.be.true();
           });
 
-          it('does not go fullscreen if roundedCorners are enabled', async () => {
+          it.skip('does not go fullscreen if roundedCorners are enabled', async () => {
```

Comment:
I understand that this test wasn't running before, but we should add a TODO comment as to why this is disabled.

Reply:
annotated in 897e7e72c1583c901e7e036063ecde4608d9be90